src/main/kotlin/domain/Symbol.kt

```diff
@@ -0,0 +1,5 @@
+package domain
+
+enum class Symbol(val char: String) {
```

Comment:
도메인의 관심사는 그 셀의 symbol이 무엇인지(어떻게 화면상에 표시되는지)가 아니라 그 `cell`이 `지뢰`인지 아닌지인 것 같아요:) `Symbol`이라는 이름은 화면상에 어떻게 보여지는지에 치중된 이름으로 느껴져서 `CellType`과 같은 이름으로 변경해보는건 어떨까요?

```suggestion
enum class CellType(val symbol: Char) {
    MINE('*'),
    NORMAL('C'),
}
```

Reply:
말씀주신 것 처럼 화면상의 노출되는 것에만 치중했던 네이밍이었네요,
디테일한 지적 감사합니다 👍